packages/cxl-ui/scss/cxl-theme.scss

```diff
@@ -0,0 +1,85 @@
+html,
+body {
+  height: 100%;
+  min-height: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.sub-title {
+  margin-top: var(--lumo-space-tall-l);
+  text-align: center;
+}
+
+.flex {
+  display: flex;
+}
+
+strong {
+  color: var(--lumo-primary-text-color);
+}
+
+.align {
+  &-left {
+    text-align: left;
+  }
+  &-center {
+    text-align: center;
+  }
+  &-right {
+    text-align: right;
+  }
+}
+
+.screen-reader-text {
+  display: none;
+}
+
+.wrap {
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: calc(var(--lumo-space-m) / 2);
+  padding-right: calc(var(--lumo-space-m) / 2);
+
+  @media screen and (min-width: 1088px) {
```

Comment:
Replace w/ master media queries defined here?

Reply:
Don't know what is the decision here. These are WP breakpoints (should use the current mq mixins though) but the breakpoints are different

Our project breakpoints:
```
small: 724px,
medium: 992px,
large: 1279px,
largehd: 1600px
```

and compared to WP .wrap breakpoints that we changed to (before it was .container)
<del>```small: 724px```</del>
```
medium: 1088
large: 1280px,
largehd: 1472px
```

Reply:
What is "WP .wrap" even? Screenshot?

I think current WP whatever doesn't matter, it will be adjusted to what we do here.